src/raygun.utilities.js

```diff
@@ -51,6 +51,84 @@ var raygunUtilityFactory = function (window) {
     };
   }
 
+  // Production steps of ECMA-262, Edition 5, 15.4.4.19
+  // Reference: http://es5.github.io/#x15.4.4.19
+  if (!Array.prototype.map) {
```

Comment:
Perhaps, I was just following the existing convention. It would make more sense, I can move em out to raygun.polyfills.js